packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.test.tsx

```diff
@@ -189,6 +231,7 @@ describe('QuestionnaireBuilder', () => {
         target: { value: 'Renamed' },
       });
     });
+    fireEvent.blur(screen.getByDisplayValue('Renamed'));
```

Comment:
Switched the textinputs from onChange to onBlur. Tested it in the medplum app and storybook. Didn't seem to change the user experience. Change change it back if needed